### PR TITLE
CS batch 20/an estimated 26

### PR DESCRIPTION
**Pull request by greg0ire**
From 699 to 554 errors.

All remaining errors are in `lib` :tada:

### CODE REVIEW

File: phpcs.xml.dist [hunk 1]
```diff
@@ -192,6 +196,9 @@
 
     <rule ref="Squiz.Commenting.FunctionComment.WrongStyle">
         <!-- https://github.com/squizlabs/PHP_CodeSniffer/issues/1961 -->
+        <exclude-pattern>tests/Doctrine/Tests/Mocks/DatabasePlatformMock.php</exclude-pattern>
+        <exclude-pattern>tests/Doctrine/Tests/Mocks/DriverMock.php</exclude-pattern>
+        <exclude-pattern>tests/Doctrine/Tests/Mocks/SequenceMock.php</exclude-pattern>
```
Comment (SenseException) [inline phpcs.xml.dist:201]: Is this because of `/* MOCK API */`? Does the WrongStyle error happen with `// MOCK API ` too?

File: phpcs.xml.dist [hunk 2]
```diff
@@ -202,9 +209,16 @@
...
     <rule ref="PSR2.Methods.MethodDeclaration.Underscore">
         <!-- extending a class from another package -->
+        <exclude-pattern>tests/Doctrine/Tests/Mocks/DatabasePlatformMock.php</exclude-pattern>
+        <exclude-pattern>tests/Doctrine/Tests/Mocks/SchemaManagerMock.php</exclude-pattern>
         <exclude-pattern>tests/Doctrine/Tests/ORM/Functional/Ticket/DDC3634Test.php</exclude-pattern>
     </rule>
 
+    <rule ref="Squiz.NamingConventions.ValidVariableName.PublicHasUnderscore">
+        <!-- the impact of changing this would be too big -->
+        <exclude-pattern>tests/Doctrine/Tests/DbalFunctionalTestCase.php</exclude-pattern>
```
Comment (SenseException) [inline phpcs.xml.dist:219]: What's the public underscored name?

**Review by SenseException — APPROVED** [review]

File: phpcs.xml.dist (hunk 2 above)
Comment (greg0ire) [reply phpcs.xml.dist:219]: `$_conn`. Although it is protected, that rule triggers.

File: phpcs.xml.dist (hunk 1 above)
Comment (greg0ire) [reply phpcs.xml.dist:201]: Yes and yes

File: phpcs.xml.dist (hunk 2 above)
Comment (greg0ire) [reply phpcs.xml.dist:219]: See here why: https://github.com/squizlabs/PHP_CodeSniffer/blob/0aeb095248f307cc7bac7b0268ae4d5d3440f300/src/Standards/Squiz/Sniffs/NamingConventions/ValidVariableNameSniff.php#L129

**Pull request merged**